pkg/probe/probes.go

```diff
@@ -124,10 +124,18 @@ func defaultGw() (string, error) {
 		if err != nil {
 			return false, errors.Wrap(err, "failed retrieving current state to retrieve default gw")
 		}
+		defaultGwGjsonPath := "routes.running.#(destination==\"0.0.0.0/0\").next-hop-address"
 		defaultGw = gjsonCurrentState.
-			Get("routes.running.#(destination==\"0.0.0.0/0\").next-hop-address").String()
+			Get(defaultGwGjsonPath).String()
 		if defaultGw == "" {
-			log.Info("default gw missing", "state", gjsonCurrentState.String())
+			msg := "default gw missing"
+			defaultGwLog := log.WithValues("path", defaultGwGjsonPath)
+			defaultGwLogDebug := defaultGwLog.V(1)
```

Comment:
I'm not sure I understand the verbosity levels - in main.go there is a debug msg with V(2), why is here `defaultWgLogDebug` with `V(1)`?

Reply:
Forgot to remove the debug trace at main.go I added it to check what level works fine, the levels are from zap library and 1 works if we are in dev mode.

Reply:
Thanks